opm/parser/eclipse/EclipseState/EclipseState.hpp

```diff
@@ -153,6 +153,12 @@ namespace Opm {
         const std::map<int, VFPInjTable>& getVFPInjTables() const;
         size_t getNumPhases() const;
 
+        // The saturation function family.
+        // If SWOF and SGOF are specified in the deck it return 1
+        // If SWFN, SGFN and SOF3 are specified in the deck it return 2
+        // If keywords are missing or mixed, an error is given.
+        const size_t getSaturationFunctionFamily() const;
```

Comment:
The 2d tables settles it for me: we cannot hope to unify all saturation specification methods. So now I find it totally reasonable that the method is private (that started this discussion)...